README.md

```diff
@@ -16,19 +16,22 @@ The lisp.New function creates a new running process with a default evaluation en
 Eval returns an sexpression and an error. Only evaluating a single sexpression is supported at the moment, but wrapping multiple expressions in `begin` serves to eval all at once.
 
 ```go
+package main
+
 import (
-    "fmt"
-    "log"
-    "github.com/deosjr/lispadventures/lisp"
+	"fmt"
+	"log"
+
+	"github.com/deosjr/whistle/lisp"
 )
 
 func main() {
-    l := lisp.New()
-    e, err := l.Eval("(* 6 7)")
-    if err != nil {
-    	log.Fatal(err)
-    }
-    fmt.Println(e) // prints 42
+	l := lisp.New()
```

Comment:
Seems go fmted then copied in, leaving indentation inconsistent. Smth about tabs/spaces in markdown idk, but further down its different

Reply:
Switching from spaces to tabs for README.

Reply:
Done.